lib/evmone/instructions.hpp

```diff
@@ -898,6 +935,22 @@ evmc_status_code create_impl(StackTop stack, ExecutionState& state) noexcept;
 inline constexpr auto create = create_impl<OP_CREATE>;
 inline constexpr auto create2 = create_impl<OP_CREATE2>;
 
+inline code_iterator callf(StackTop /*stack*/, ExecutionState& state, code_iterator pos) noexcept
+{
+    const auto index = (size_t{pos[1]} << 8) | pos[2];
```

Comment:
Can we add a `read16be` helper into this file? It is used in every new instruction.

Reply:
It's also needed in `eof.cpp`. I will define proper common helper in some header.